metsim/constants.py

```diff
@@ -0,0 +1,56 @@
+"""
+Stores default constants, parameters, and options.  
+"""
```

Comment:
I'm not sure which things should be considered tunable, but they should all go in the MetSim parameters dictionary instead of the constants module, so they can be specified in the configuration file.